defaults.env

```diff
@@ -1,5 +1,5 @@
 export CORECLR_ENABLE_PROFILING=1
 export CORECLR_PROFILER='{918728DD-259F-4A6A-AC2B-B85E1B658318}'
-export CORECLR_PROFILER_PATH=/opt/opentelemetry-dotnet-autoinstrumentation/OpenTelemetry.AutoInstrumentation.ClrProfiler.Native.so
+export CORECLR_PROFILER_PATH=/opt/opentelemetry-dotnet-autoinstrumentation/OpenTelemetry.ClrProfiler.Native.so
```

Comment:
```suggestion
export CORECLR_PROFILER_PATH=/opt/opentelemetry-dotnet-clrprofiler/OpenTelemetry.ClrProfiler.Native.so
```

Reply:
Separate PR seems good idea. I'll add to todo list.
Found references to different confusing paths `/opt/otel-dotnet-autoinstrumentation/defaults.env`, `/opt/opentelemetry-dotnet-clrprofiler/...`, `/opt/datadog/` ... so seems like needs bigger investigation whats in use and what should be the actual one.